libglusterfs/src/glusterfs/stack.h

```diff
@@ -289,10 +289,8 @@ get_the_pt_fop(void *base_fop, int fop_idx)
                      frame->root, old_THIS->name, THIS->name);                 \
         /* Need to capture counts at leaf node */                              \
         if (!next_xl->pass_through && !next_xl->children) {                    \
-            GF_ATOMIC_INC(next_xl->stats.total.metrics[opn].fop);              \
-            GF_ATOMIC_INC(next_xl->stats.interval.metrics[opn].fop);           \
-            GF_ATOMIC_INC(next_xl->stats.total.count);                         \
-            GF_ATOMIC_INC(next_xl->stats.interval.count);                      \
+            GF_ATOMIC_INC(next_xl->stats[opn].total_fop);                      \
```

Comment:
I *think* we can remove the totals as well (calculate them from all the intervals), but we should do it in a separate patch.
(**https://github.com/gluster/glusterfs/issues/1583#issuecomment-706052042** mentions it - and we probably need a separate issue for it as well)

Reply:
will send a separate patch to use a single variable.